Crm/PersonAttributeForms.ascx.cs

```diff
@@ -1037,6 +1052,22 @@ private void tfeForm_DeleteFieldClick( object sender, AttributeFormFieldEventArg
                 var field = form.Fields.FirstOrDefault( f => f.Guid == e.FormFieldGuid );
                 if ( field != null )
                 {
+                    /*
+                      On Field Delete, we need to also remove all the existing reference of current field in filter rule list
+                    */
+                    var newFormFieldsWithRules = form.Fields
```

Comment:
Test functionality within 13 to ensure nothing changed

Reply:
Tested in v13 and did a quick cursory review of some of the code, doesn't look like it changed at all.